helm/alfresco-process-infrastructure/values.yaml

```diff
@@ -159,7 +159,7 @@ alfresco-modeling-app:
     path: /modeling
   image:
     repository: quay.io/alfresco/alfresco-modeling-app
-    tag: 7.1.0-M7
+    tag: "development"
```

Comment:
tag is now develop everywhere including front-end

Reply:
perfect, changing it now